### ISSUE TITLE
Failed Touch-off has no remediation plan

### ISSUE THREAD

**Issue by mbromley-voxel** [mbromley-voxel/ECM-Control-System#69]
Touch-offs commonly fail. Currently there is no way to address this in the configurations, it just stops the execution of the configuration, but there should be some method to retry the touch off.

**Comment by majorpr13** [comment]
We can add a state, similar to the pause state on the touchoff asking the user how they want it handled. Assuming same parameters would be used to execute the touchoff.

**Comment by mbromley-voxel** [comment]
Yes we want to rerun with the same parameters.  Often times it will fail for inconsistency and just running it again will correct the issue.
